backend/src/main/resources/application.yaml

```diff
@@ -20,4 +20,9 @@ v2:
     sendingApplication: HNWeb
     sendingFacility: HN-WEB
     receivingFacility: BC00001013
-    processingID: D
\ No newline at end of file
+    processingID: D
+
+# RAPID fixed width message construction
+rapid:
+  header: 
+    organization: "00000010"
```

Comment:
I think this might actually need to come from the token, the same way as we've done for V2

Reply:
I updated the converter to use the existing code to grab it from the token. There will be some further changes in the R41 branch to remove the RapidDefaults code entirely.